app/assets/javascripts/discourse/app/components/composer-container.gjs

```diff
@@ -41,14 +41,19 @@ export default class ComposerContainer extends Component {
       @cancelled={{this.composer.cancelled}}
       @save={{this.composer.saveAction}}
     >
-      <div class="grippie"></div>
+      {{#unless this.site.mobileView}}
+        <div class="grippie"></div>
+      {{/unless}}
```

Comment:
My only concern with this is tablets. IIRC, some default to a mobile view on Android?